### DISCUSSION TITLE
How to add a subtitle to the report?

### DISCUSSION THREAD

**Discussion by Coder2023Coder** [pytest-dev/pytest-html#633] [Q&A]
I want to add a subtitle to the report after the title. I am showing where I mean by a red arrow in the attached image. 
I know that to change the title I should add the following to the `conftest.py`:
```
def pytest_html_report_title(report):
    report.title = "<new title>"
```
But with a subtitle I wonder how to add it.

Also, is it possible to modify the content of the line "Report generated on ... "?

![Screenshot from 2023-04-07 16-27-39](https://user-images.githubusercontent.com/129788129/230635503-31cecd1d-e833-4689-a9a8-4318a307cefe.png)

**Comment by BeyondEvil** [comment]
Currently there's no way of adding a subtitle or altering the "Report generated..." line.

What's the use case?

The closest you can get is using the [pytest_html_results_summary](https://pytest-html.readthedocs.io/en/latest/user_guide.html#additional-summary-information) hook (prefix).
